features/admin/invite_supplier_contributor.feature

```diff
@@ -4,10 +4,10 @@ Feature: Invite a contributor to a supplier account
 Scenario Outline: Correct users can invite a contributors to a supplier account
   Given I am logged in as the production <role> user
   And I have a supplier with:
-    | name          | DM Functional Test Supplier 1 |
+    | name          | Invite a contributor feature DM Functional Test Supplier |
```

Comment:
I think we should do this for the other two admin supplier scenarios (`deactivate_supplier_user` and `update_supplier_name`) as well, to avoid future tests succumbing to the same problem.

I'd prefer to keep `DM Functional Test..` at the beginning of the string in case we do need to easily identify all the test accounts - would `DM Functional Test Invite A Contributor Supplier` still work?

Reply:
That's a smashing point. I'll update.